CMakeLists.txt

```diff
@@ -363,17 +364,17 @@ elseif(UNIX)
   list(APPEND mxnet_LINKER_LIBS pthread)
 endif()
 
+
 # ---[ LAPack
-if(USE_LAPACK AND NOT MSVC)
+if(USE_LAPACK)
   add_definitions(-DMXNET_USE_LAPACK=1)
-  list(APPEND mxnet_LINKER_LIBS lapack)
-else(USE_LAPACK)
-  # Workaround for Windows until using new Jenkinsfile.
-  if(BLAS STREQUAL "Open" OR BLAS STREQUAL "open" OR USE_BLAS STREQUAL "Open" OR USE_BLAS STREQUAL "open")
-    add_definitions(-DMXNET_USE_LAPACK=1)
+  if (NOT MSVC)
+    list(APPEND mxnet_LINKER_LIBS lapack)
```

Comment:
There is no separate liblapack for openblas as far as I understood

Reply:
I don't get your comment, I tried to fix the logic as I understood the previous logic was working around adding these flags in the windows case and breaking other platforms. Can you elaborate?

Reply:
`list(APPEND mxnet_LINKER_LIBS lapack)` line adds liblapack to required linker libraries. But there is no such think as liblapack for OpenBLAS, at least not for our builds. The CI passed in this case because with this change `-DUSE_LAPACK=OFF` becomes effective.

Reply:
And there is a chance that liblapack exists for other blas libraries (atlas?) but we are not testing them in CI currently

Reply:
Just create symlinks such as libcblas and liblapack to the compiled libopenblas. There’s no need to hack around.

Reply:
Ok, this might be an option

Reply:
Thanks for the clarification, I got now what was the concern. We need better ARM / ANDROID architecture detection when cross compiling in CMake. This might be something we can definitely refine later. For the moment I'm fine with either solution proposed by @lebeg  or @szha I think we will need to refine and improve this logic on future PRS.